client/app.js

```diff
@@ -1,9 +1,12 @@
 import React, {Component} from 'react'
 import {Navbar} from './components'
+import SideNavbar from './components/sideNav'
 import Routes from './routes'
 //import { connect } from 'http2';
```

Comment:
I know you didn't add this, but could you remove this once and for all before your merge?